src/main/java/io/sniffy/Sniffer.java

```diff
@@ -26,19 +32,21 @@ private Sniffer() {
 
     // Registered listeners (i.e. spies)
 
+    private final static AtomicInteger executedStatementsGlobalCounter = new AtomicInteger();
+
     private static final List<WeakReference<Spy>> registeredSpies = new LinkedList<WeakReference<Spy>>();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/sniffy/sniffy/file/2495648593/issues/source?bid=3005960&fileBranchId=3034483#l37)